TVOAppliance/hf_upgrade.sh

```diff
@@ -95,6 +95,31 @@ function reconfigure_s3_service_path()
 	sed  -i "s~$src_string~$dest_string~g" $file_name
 
 }
+
+#function to keep only tls1_2 access
+function limit_access_to_tls1_2()
+{
+	config_file="/etc/nginx/nginx.conf"
+	config_line="ssl_protocols TLSv1.2;"
+
+	# Check if the line already exists in the file
+	if grep -q "$config_line" "$config_file"; then
+		echo "Line already exists in $config_file"
```

Comment:
this output msg won't make any sense to the user.
Mention below msg instead:
if already exists, "SSL protocol set to TLSv1.2 already"
else, "Setting the SSL protocol to TLSv1.2"